### ISSUE TITLE
Which version should we use, master vs 1.2.0?

### ISSUE THREAD

**Issue by mhewedy** [tiaguinho/gosoap#41]
Which version should we use, master vs 1.2.0?

And whether the README example works with the v1.2.0 (that by default got grabbed?)... I think it requires modifications...

In other words, is it safe to do `go get github.com/tiaguinho/gosoap@master`?

**Comment by xhit** [comment]
I use master in production and works like expected.

The master version have 37 commits ahead with a lot of bug fixes.

**Comment by tiaguinho** [comment]
The master will always have the last version of the package.
You can watch the package to see when a new version is released and check if it will break something.

**Comment by l3uddz** [comment]
@tiaguinho able to make v2.1.1 compatible with go modules? i cannot get it to work properly using `github.com/tiaguinho/gosoap/v2 v2.1.1` in my go.mod

**Comment by tiaguinho** [comment]
Of course.
Sorry about that.
Can you check now @l3uddz ?

Thank you!

**Comment by l3uddz** [comment]
@tiaguinho Im unable to get it working with v2.1.1, see below:

```
➜  git:(develop) ✗ go get github.com/tiaguinho/gosoap@v1.2.0
➜  git:(develop) ✗ go get github.com/tiaguinho/gosoap@v2.1.1
go get github.com/tiaguinho/gosoap@v2.1.1: github.com/tiaguinho/gosoap@v2.1.1: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
➜  git:(develop) ✗
```

**Comment by tiaguinho** [comment]
@l3uddz I did not start using v2 as I should be. So, to fix that I change the version to v1.4.1.
Sorry for my mistake.

**Comment by l3uddz** [comment]
@tiaguinho thank you, it is now working 😄
